spec/factories/account.rb

```diff
@@ -17,5 +18,13 @@
         }.to_json)
       )
     end
+
+    transient do
+      auth_type { nil }
+    end
```

Comment:
Short-enough to be a one-liner:
```suggestion
    transient { auth_type { nil } }
```